src/adapters/IDB.js

```diff
@@ -142,6 +142,27 @@ const cursorHandlers = {
   },
 };
 
+const transformSubObjectFilters = filtersObj => {
+  const newFilter = {};
+  const makeNestedObjectFromArr = (arr, val) => {
+    const last = arr.length - 1;
+    arr.reduce((acc, cv, i) => {
+      return i === last
+        ? (acc[cv] = val)
```

Comment:
It's not a recommended practice to put assignments in ternary operators.

```js
if (i === last) {
  acc[cv] = val;
} else if ...
```
is better